setup.py

```diff
@@ -208,13 +208,12 @@ def get_git_sha() -> str:
         "starrocks": ["starrocks>=1.0.0"],
         "doris": ["pydoris>=1.0.0, <2.0.0"],
     },
-    python_requires="~=3.9",
+    python_requires="~=3.9",  # TODO: change it to "~=3.10"
```

Comment:
I missed this just now, can we make it 
```suggestion
    python_requires="~=3.10",
```